pixie/stdlib.pxi

```diff
@@ -2110,3 +2109,16 @@ Expands to calls to `extend-type`."
                 `(-dispose! ~nm))
               names)
        result#)))
+(defn pst
+  {:doc "Prints the trace of a Runtime Exception if given, or the last Runtime Exception in *e"
+   :signatures [[] [e]]
+   :added "0.1"}
+  ([] (pst *e))
+  ([e] (when e (ex-pr e))))
```

Comment:
You can already do this with `(print (str e))`, then you don't need a separate `ex-pr` or alternatively implement it as just `(print (str e))`.

(Sorry that this is so late.)